lib/vagrant-cloudstack/action/run_instance.rb

```diff
@@ -34,7 +35,7 @@ def call(env)
           @network          = CloudstackResource.new(domain_config.network_id, domain_config.network_name, 'network')
           @service_offering = CloudstackResource.new(domain_config.service_offering_id, domain_config.service_offering_name, 'service_offering')
           @disk_offering    = CloudstackResource.new(domain_config.disk_offering_id, domain_config.disk_offering_name, 'disk_offering')
-          @template         = CloudstackResource.new(domain_config.template_id, domain_config.template_name, 'template')
+          @template         = CloudstackResource.new(domain_config.template_id, domain_config.template_name || env[:machine].config.vm.box, 'template')
```

Comment:
Just created a new VM, installed Vagrant plugin and all stuff.
Created a Vagrantfile without the box_url, "up"-ed the VM without problems and without requiring or downloading a "dummy" box...

Was triggered by https://github.com/mitchellh/vagrant-rackspace/blame/master/README.md#L52 .